walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -23,128 +23,129 @@
 import java.util.Set;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-
-    Collection<String> requiredAttendees = request.getAttendees();
-    Collection<String> optionalAttendees = request.getOptionalAttendees();
-    Collection<TimeRange> blockedTimes = new HashSet<>();
-    Collection<TimeRange> optionalBlockedTimes = new HashSet<>();
-    Collection<TimeRange> timesToAdd = new HashSet<>();
-    Collection<TimeRange> optionalTimesToAdd = new HashSet<>();
-    
+  public Collection < TimeRange > query(Collection < Event > events, MeetingRequest request) {
+
+    Collection < String > requiredAttendees = request.getAttendees();
+    Collection < String > optionalAttendees = request.getOptionalAttendees();
+    Collection < TimeRange > blockedTimes = new HashSet < > ();
+    Collection < TimeRange > timesToAdd = new HashSet < > ();
+    Collection < Event > optionalEvents = new HashSet < > ();
+
+    // EDGE CASE
     if ((requiredAttendees.isEmpty()) && (request.getDuration() < TimeRange.WHOLE_DAY.duration())) {
-        if (!optionalAttendees.isEmpty()) {
-            timesToAdd.add(TimeRange.WHOLE_DAY);
-        }
+      if (!optionalAttendees.isEmpty()) {
+        timesToAdd.add(TimeRange.WHOLE_DAY);
+      }
     }
 
     // For each event, check if anyone attending the request is also attending that event.
-    for (Event event : events) {
-        Set<String> intersection = new HashSet<String>(requiredAttendees);
-        Set<String> optionalIntersection = new HashSet<String>(optionalAttendees);
-        intersection.retainAll(event.getAttendees());
-        optionalIntersection.retainAll(event.getAttendees());
-
-        // If one or more requiredAttendees are attending this event, then add it to blockedTimes
-        // Otherwise, if any optional attendees are attending this event, then add it to optionalBlockedTimes
-        if (!intersection.isEmpty()) {
-            blockedTimes.add(event.getWhen());
-        } else if (!optionalIntersection.isEmpty()) {
-            optionalBlockedTimes.add(event.getWhen());
-        }
+    for (Event event: events) {
+      Set < String > intersection = new HashSet < String > (requiredAttendees);
```

Comment:
Are these spaces across files were made automatically? They should not... Set<string> instead of Set < string > name

Reply:
Yeah I think the formatter I use makes Set <String> become Set < String >, I'll fix that!

Reply:
I think the formatter I use makes Set <String> become Set < String >. Ill fix that!